openshift-hack/images/Dockerfile.openshift

```diff
@@ -1,9 +1,11 @@
 # Build vpcctl binary
 FROM registry.ci.openshift.org/ocp/builder:rhel-8-golang-1.16-openshift-4.9 as vpc-builder
 WORKDIR /build
-RUN git clone https://github.com/openshift/cloud-provider-vpc-controller && \
-    cd cloud-provider-vpc-controller && \
-    make build-linux
+COPY vendor/cloud.ibm.com/cloud-provider-vpc-controller .
+COPY openshift-hack/images/go.mod .
+RUN go mod tidy && \
+    go mod vendor && \
```

Comment:
This will pull in dependencies from the internet won't it? I think we just need to copy the whole vendor folder in and then complete the build right?

Reply:
Agree, it makes more sense!